README.md

```diff
@@ -24,65 +40,101 @@ The structure of this lab — where its files and folders are located — looks
+All labs will more or less have the same structure. (And non-lab lessons, for
+that matter, will still have CONTRIBUTING.md, LICENSE.md, and README.md files.)
+
+### Code Along
 
-### Code-along
-For now, open up `index.js` in your text editor. If you're using the Learn IDE, click the blue "Open" button in the top right hand corner of the lesson. If you open up that `js-basics-variables-lab/` directory, you'll see a list of files (along with a `test/` directory). Click `index.js`, and it will open in the editor.
+Open up `index.js` in your text editor. If you're using the Learn IDE, click the
+blue "Open" button in the top right hand corner of the lesson. If you open up
+that `js-basics-variables-lab/` directory, you'll see a list of files (along
+with a `test/` directory). Click `index.js`, and it will open in the editor.
 
 In `index.js`, you should see, well, nothing. We'll fix that soon.
 
-Now open up `test/indexTest.js`. Hey, there's something! What's all of this stuff doing?
+Now open up `test/indexTest.js`. Hey, there's something! What's all of this
+stuff doing?
 
-**Note: The `test/indexTest.js` has great info that we want to look at, but do not edit this file otherwise you may have extra difficulty passing this lab.**
+**Note:** The `test/indexTest.js` has great info that we want to look at, but do
+not edit this file otherwise you may have extra difficulty passing this lab.
 
 A few lines down in the `test/indexTest.js` file you will see:
+
 ```js
 describe('index.js', function () {
   // there's stuff in here, too
 });
 ```
 
-`describe` is a function provided by our test library, Mocha. The word is basically used to hold our tests. You can see that after the word `describe` is a description about our tests. Tests are used as a way to document the behavior of a function to developers, and you can see that examples of that after the words `describe`. For example, the next word `describe` is followed by the word `companyName` name. Here the test is telling us that the tests that come after words will be about `companyName`. Then comes the word `it`, where you see the following:
+`describe` is a function provided by our test library, Mocha and it's used to
+hold our tests. After the word `describe` is information about our tests. Tests
+are used as a way to document the behavior of a function to developers. For
+example, the next word `describe` is followed by the word `companyName` name.
+Here the test is telling us that the tests that come after words will be about
+`companyName`. Then comes the word `it`, where you see the following:
+
 ```js
 it('is set as Scuber', function () {
   // tests are here
 });
 ```
 
-So this is telling us that the `companyName` should be set to `Scuber`. Finally, filling in the missing part of the `it` code, we see:
+So this is telling us that the `companyName` should be set to `Scuber`. Finally,
+filling in the missing part of the `it` code, we see:
```

Comment:
We start a lot of sentences with "so". That's fine sometimes, but I think when reading a technical document like this, it starts to sound a lot like a lot of run on sentences that become hard to mentally parse without reading it or hearing it out loud.

> In this example, we see that the `companyName` should be set to `Scuber`. The last piece of this code for the `it` block of code, we see: